src/NuGetPackageVerifier/PackageIssueFactory.cs

```diff
@@ -209,5 +209,15 @@ public static PackageVerifierIssue AssemblyHasWrongJsonNetVersion(string assembl
                 string.Format("The assembly '{0}' references the wrong Json.NET version. Current version '{1}'; Expected version '{2}'.", assemblyPath, currentVersion, expectedVersion),
                 PackageIssueLevel.Error);
         }
+
+        public static PackageVerifierIssue IdDoesNotExist(string id)
+        {
+            return new PackageVerifierIssue("PACKAGE_OWNERSHIP", id, $"The id '{id}' does not exist on NuGet.org.", PackageIssueLevel.Warning);
```

Comment:
Can we add something to these messages saying what to do in case of failure? The fix is of course not obvious. Perhaps a URL to a wiki page on Coherence-Signed or something?